### DISCUSSION TITLE
Jetbot controlled locally

### DISCUSSION THREAD

**Discussion by NikolaAndro** [NVIDIA-AI-IOT/jetbot#482] [Q&A] [answered]
Is it possible to run the live_demo code on local machine?

I get:

ERROR: Could not find a version that satisfies the requirement jetbot (from versions: none)
ERROR: No matching distribution found for jetbot

I do not know how to import module jetbot!

Thank you!

**Comment by JHPHELAN** [comment]
@NikolaAndro have you solved your question?
If not, can you provide a link to the live_demo you're trying to run (there are several).

> I do not know how to import module jetbot!

Seems you're having a problem with the basic Jetbot install.  Try again from the beginning.

**Comment by NikolaAndro** [comment, marked as answer]
Hello @JHPHELAN,

No, I was not able to recreate the jetbot environment on the local machine. The jetbot module is I guess not available anywhere but on the jetbots. 

My goal was to train models on the local machine since it is much more powerful and then transfer the model to the jetbot. However the dependency issue remained unsolved.

**Reply by JHPHELAN** [reply]
@NikolaAndro 
I think you'll find the Jetbot with its Nano is quite powerful and designed for AI model training.  In the collision_avoidance project I tried, it trained the module in only about a minute!  Suggest doing your training on the Jetbot as the example suggests.  Keep us posted of your progress.
Jim

**Reply by NikolaAndro** [reply]
@JHPHELAN  Thank you for your response. I will keep doing it that way.

Nikola
